library/common/extensions/filters/http/platform_bridge/c_types.h

```diff
@@ -24,11 +24,10 @@ typedef struct {
 /**
  * Return codes for on-data filter invocations. @see envoy/http/filter.h
  */
-typedef enum {
-  ENVOY_FILTER_DATA_STATUS_CONTINUE = 0,
-  ENVOY_FILTER_DATA_STATUS_STOP_ITERATION_AND_BUFFER,
-  ENVOY_FILTER_DATA_STATUS_STOP_ITERATION_NO_BUFFER,
-} envoy_filter_data_status_t;
+typedef int envoy_filter_data_status_t;
```

Comment:
Can we document this style decision in STYLE.md?

Reply:
Sure, but just to clarify, which do you mean?

Reply:
The new approach with the static cast

Reply:
Oh, sure.

Reply:
Maybe as part of this PR?
https://github.com/lyft/envoy-mobile/pull/966

Reply:
will document in https://github.com/lyft/envoy-mobile/pull/966/files